SU2_CFD/include/numerics/CNumerics.hpp

```diff
@@ -34,6 +34,7 @@
 #include <cstdlib>
 
 #include "../../../Common/include/CConfig.hpp"
+#include "../variables/CNEMOEulerVariable.hpp"
```

Comment:
I believe this include wont be necessary with the move to fluid model.

Reply:
with the move to fluidmodel many unnecessary/redundant things will be removed yes, but for now it's necessary for things to work